### ISSUE TITLE
Scrollback buffers // scrollable regions

### ISSUE THREAD

**Issue by tomek-szczesny** [tomek-szczesny/notcurses-vt-proto#5]
This is probably of the least concern right now, but would be handy to implement that as well. (Not a mandatory feature of a terminal after all). Also, how the hell a user is supposed to interact with it?

But even if, should Alternative buffer have scrollback capability as well?

**Comment by o-sdn-o** [comment]
> Alternative buffer have scrollback capability as well?

Alternative buffer does not have a scrollback buffer. Although some terminals have this option. For example, Apple Terminal.

**Comment by o-sdn-o** [comment]
> how the hell a user is supposed to interact with it?

This allows the user to have two independent contexts
- context with a scrollbuffer
- static frame context

**Comment by tomek-szczesny** [comment]
Right, thanks. I wasn't sure about the w rollback buffer in alternative mode.
I need to define a way the user of notcurses library could scroll through it. Probably a function or something, but surely won't work with just any ncplane. I'm starting to think VT maybe will end up as a widget instead.

**Comment by tomek-szczesny** [comment]
Related stuff: Scrollable region / area.
